src/txn/txn.c

```diff
@@ -196,7 +194,7 @@ __txn_get_snapshot_int(WT_SESSION_IMPL *session, bool publish)
     WT_TXN_GLOBAL *txn_global;
     WT_TXN_SHARED *s, *txn_shared;
     uint64_t commit_gen, current_id, id, prev_oldest_id, pinned_id;
-    uint32_t i, n, session_cnt;
+    uint32_t n;
```

Comment:
Should we rename this to `snapshot_index`?

Reply:
Renamed in [a701b04](https://github.com/wiredtiger/wiredtiger/pull/9296/commits/a701b040e72b8635be02722ae9e533fd0ff75676)